web.py

```diff
@@ -109,12 +112,6 @@ def _filter_entry_ids(self, entries):
         return result
 
 
-app = flask.Flask('plugincompat')
-app.debug = True
```

Comment:
Oh my

Reply:
yes still have to remove that from main, actually